app/views/news/2018-03-05-february-2018-monthly-update.html.md

```diff
@@ -0,0 +1,71 @@
+---
+
+Hello and welcome to the monthly update! During February, our work was supported by [Stripe](https://stripe.com), [GitLab](https://about.gitlab.com/), [reinteractive](https://reinteractive.com/), and many others.
+
+## ruby together news
+
+
+In February, Ruby Together was supported by 76 different companies, including Sapphire member [Stripe](https://stripe.com). Four companies joined as new members, including [Nebulab](https://nebulab.it/), [weLaika](https://dev.welaika.com/en/), [Kickass Partners](http://kickass.partners).
+
+In addition to those companies, 6 new developers signed up as members or friends of Ruby Together, including Stan Lo. In total, we were supported by 91 individual members and 68 friends of Ruby Together. Thanks to all of our members for making everything that we do possible. &lt;3
+
+Ruby has been accepted as a project for [Google Summer of Code 2018](https://summerofcode.withgoogle.com/)! Student project applications will open on March 12. Until then, we’re looking for mentors for students who want to work on Ruby open source projects this summer. If you are interested in mentoring a student this summer, [get in touch](mailto:hello@rubytogether.org)!
+
+For more information about what it’s like to be a mentor, Google has written a [mentor guide](https://google.github.io/gsocguides/mentor/).
+
+We would also love contributions to the project ideas list. If you have an open source project or a feature idea that are one-student-summer sized, [add them here](https://github.com/rubygsoc/rubygsoc/wiki/Ideas-List).
+
+## bundler news
+
+Bundler saw some fixes in February thanks to contributors [@nholden](https://github.com/nholden), [@voxik](https://github.com/voxik), [@cpgo](https://github.com/cpgo),  [@deivid-rodriguez](https://github.com/deivid-rodriguez), and [@alyssais](https://github.com/alyssais). Fixes this month included [a fix for certain instances of the “perhaps the lockfile is corrupted?” error](https://github.com/bundler/bundler/pull/6288), as well as [clear enforcement steps for the code of conduct](https://github.com/bundler/bundler/pull/6283). We also received [an awesome proposal](https://github.com/bundler/rfcs/pull/10) to give the `add` command superpowers, and add a `remove` command.
+
+This month, Bundler gained 23 new commits, contributed by 8 authors. There were 291 additions and 59 deletions across 50 files.
+
+
+## rubygems.org news
+
+There were no changes to RubyGems.org in February.
+
+## rubygems news
+
+RubyGems saw a lot of activity this month, including the release 2.7.5 and 2.7.6 with _tons_ of bug fixes, as well as [finally writing out the Ruby version support and release policies](https://github.com/rubygems/rubygems/pull/2202). There was also the [drafting of a minimally disruptive plan for RubyGems 3 and 4](https://github.com/rubygems/rubygems/pull/2182#issuecomment-364631805).
```

Comment:
might also want to mention a bunch of security-related fixes? https://blog.rubygems.org/2018/02/15/2.7.6-released.html